packages/manager/modules/pci/src/projects/project/kubernetes/components/node-pool/node-pool.controller.js

```diff
@@ -0,0 +1,19 @@
+import find from 'lodash/find';
+
+import {
+  EXCLUDE_FLAVOR_CATEGORIES,
+  NODE_MIN_COUNT,
+  NODE_MAX_COUNT,
+} from './node-pool.constants';
+
+export default class FlavorsListController {
+  $onInit() {
+    this.nodeMinCount = this.nodeMinCount || NODE_MIN_COUNT;
+    this.nodeMaxCount = this.nodeMaxCount || NODE_MAX_COUNT;
+    this.excludeCategories = EXCLUDE_FLAVOR_CATEGORIES;
+
+    if(!this.defaultFlavor && this.flavors) {
+      this.defaultFlavor = find(this.flavors, { name: 'b2-7' });
```

Comment:
NP: no other way to select the default flavor ? What's happen if `b2-7` is removed ?